[BUG] `make types` doesn't specify `mypy` path

# Bug Details :warning::bug:

## Description

`make types` does not specify the path to `mypy`, and as such may run a version of `mypy` which is not associated with the virtual environment. This can lead to `mypy` errors for example if modules are not installed in the environment that mypy exists in.

## Steps to Reproduce

- [ ] Is the bug reproducible?

1. In a fresh shell in `across-data-ingestion` directory, run `make types`.
2. See errors.

### Expected behavior

`mypy` runs without error.

### Actual behavior

`mypy` gives errors like this:
```python
mypy
across_data_ingestion/core/config.py:1: error: Cannot find implementation or library stub for module named "pydantic_settings"  [import-not-found]
across_data_ingestion/core/config.py:1: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
across_data_ingestion/tasks/example/example_task.py:4: error: Cannot find implementation or library stub for module named "fastapi_utils.tasks"  [import-not-found]
Found 2 errors in 2 files (checked 22 source files)
make: *** [types] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -183,7 +183,7 @@ lint: ## Run linting
 	@$(VENV_BIN)/pre-commit run;
 
 types: ## Run type checks
-	mypy
+	@$(VENV_BIN)/mypy;
 
 # Group: Running
 run: ## Run the containers

